2dcontext/pixel-manipulation/2d.imageData.create2.double.html

```diff
@@ -1,28 +1,30 @@
 <!DOCTYPE html>
 <!-- DO NOT EDIT! This test has been generated by tools/gentest.py. -->
```

Comment:
I don't see an update to this tool. Am I missing something or is this direct modification of these tests the wrong way to go?